sdk/postgresql/Azure.ResourceManager.PostgreSql/src/autorest.md

```diff
@@ -14,13 +14,13 @@ modelerfour:
 
 batch:
   - tag: package-2020-01-01
-  - tag: package-flexibleserver-2021-06
+  - tag: flexibleserver-2022-03-privatepreview
```

Comment:
The https://github.com/Azure/azure-rest-api-specs/pull/20212  was merged 20 mins ago, do you need to update the sdk accordingly？